furms-web-ui/src/main/resources/messages_en.properties

```diff
@@ -108,6 +108,11 @@ view.site-admin.settings.form.button.save=Save
 view.site-admin.settings.form.error.unexpected=Unexpected error during update Site Settings. \
                                  Contact with next level administrators for more information
 
+view.site-admin.administrators.title=Site administrators
+view.site-admin.administrators.invite.error.validation.field.invite=Wrong email
```

Comment:
Can we be more precise here? I.e. 
  User with email ___ does not exist
or sth like that?